lib/gds_api/rummager.rb

```diff
@@ -4,17 +4,32 @@
 module GdsApi
   class Rummager < Base
 
+    # Unified search
+    #
```

Comment:
Good point, added:

https://trello.com/c/eqtFozNE/281-replace-unified-search-with-search-in-the-rummager-api-client